app/init.php

```diff
@@ -1161,7 +1161,7 @@ function (mixed $value) {
         ->setMetadata('host', \gethostname())
         ->setMetadata('project', 'console')
         ->setTimeout(APP_DATABASE_TIMEOUT_MILLISECONDS);
-
```

Comment:
Without this initial load was always getting to pool empty

Reply:
I looked at the PR. Im okay with all the change except this one. There can be unexpected side effects. 
You can also tweak the values of these variables only for the test in cloud 
https://github.com/appwrite/appwrite/blob/main/app/init.php#L790-L805

Reply:
reverted this.